File: src/Example3Java/EX19.java
package Example3Java;

import java.util.Arrays;
import java.util.Scanner;

public class EX19 {
    public static void main(String[] args) {
        System.out.print("Enter x : ");
        int x = new Scanner(System.in).nextInt();
        System.out.print("Enter y : ");
        int y = new Scanner(System.in).nextInt();
        int[] num = {1, 2, 3, 4, 5, 6, 7, 8, 9, 10};
        int[] snum = new int[y - x + 1];
        int index = 0;
        for (int i = x; i < y + 1; i++) {
            snum[index] = num[i];
            index++;
        }
        System.out.println("Array num : " + Arrays.toString(num));
        System.out.println(Arrays.toString(snum));
    }
}

File: src/Example3Kotlin/EX19.kt
import java.util.*

object EX19 {
    @JvmStatic fun main(args: Array<String>) {
        print("Enter x : ")
        val x = Scanner(System.`in`).nextInt()
        print("Enter y : ")
        val y = Scanner(System.`in`).nextInt()
        val num = intArrayOf(1, 2, 3, 4, 5, 6, 7, 8, 9, 10)
        val snum = IntArray(y - x + 1)
        var index = 0
        for (i in x..y + 1 - 1) {
            snum[index] = num[i]
            index++
        }
        println("Array num : ${num.contentToString()}")
        println(snum.contentToString())
    }
}
